packages/flutter_availability/lib/src/ui/widgets/base_page.dart

```diff
@@ -45,7 +45,10 @@ class BasePage extends StatelessWidget {
             ),
             child: Align(
               alignment: Alignment.bottomCenter,
-              child: Column(children: buttons),
+              child: Column(
```

Comment:
wouldn't removing the column produce the same result?

Reply:
it can't be removed because Align doesn't accept a list of widgets